src/bin/tree_model_sort.rs

```diff
@@ -6,8 +6,7 @@ extern crate gdk;
 extern crate gio;
 extern crate gtk;
 
-use gio::ApplicationExt;
-use gio::ApplicationExtManual;
```

Comment:
This change is good nonetheless. We should not import traits manually but the prelude.